platform/README.md

```diff
@@ -71,9 +73,11 @@ Add configuration in `local.settings.json`
 ```
 
 ### Running tests
-
+[comment]: <> (Are there any other tests for the APIs? Should we document at a high level what the tests are doing?)
```

Comment:
This is covering specifically in the testing documentation.  We should avoid duplication, otherwise will have a maintenance nightmare.